metaflow/plugins/airflow/airflow.py

```diff
@@ -373,7 +373,7 @@ def _to_job(self, node):
             "METAFLOW_AZURE_STORAGE_BLOB_SERVICE_ENDPOINT"
         ] = AZURE_STORAGE_BLOB_SERVICE_ENDPOINT
         env["METAFLOW_DATASTORE_SYSROOT_AZURE"] = DATASTORE_SYSROOT_AZURE
-        env["METAFLOW_DATASTORE_CARD_AZUREROOT"] = DATASTORE_CARD_AZUREROOT
+        env["METAFLOW_CARD_AZUREROOT"] = CARD_AZUREROOT
```

Comment:
The same applies here too. If we already `propagate` then why not use the `config_values` function to get all the appropriate config values.

Reply:
Same response as above :).